### DISCUSSION TITLE
Project arhitecture

### DISCUSSION THREAD

**Discussion by alexradunet** [StereoKit/StereoKit#282] [General]
What are some good arhitecture tips that you can use in a new StereoKit.

I'm a backend .net developer and I'm wondering how I can apply different arhitectures to make building a StereoKit application as easy and readable as possible.

Thanks

**Comment by TopperDEL** [comment]
You should be able to create apps exactly like you do with "normal" frontends. You can create View Models (e.g. with MvvmGen) and use Dependency Injection with DI containers etc. I had the same "problem" with my first StereoKit project but quickly realized that it can be done quite similar.
Does that help you?
